tasks/DroneFlightControlTask.cpp

```diff
@@ -330,6 +338,28 @@ void DroneFlightControlTask::takeoff(VehicleSetpoint setpoint)
     mVehicle->control->takeoff(mFunctionTimeout);
 }
 
+void DroneFlightControlTask::reactiveTakeoffVelocity(VehicleSetpoint setpoint)
+{
+    auto djiStatusFlight =
+        mVehicle->subscribe->getValue<Telemetry::TOPIC_STATUS_FLIGHT>();
+    auto djiDisplayMode =
+        mVehicle->subscribe->getValue<Telemetry::TOPIC_STATUS_DISPLAYMODE>();
+
+    if (djiDisplayMode == DisplayMode::MODE_ASSISTED_TAKEOFF ||
+        djiDisplayMode == DisplayMode::MODE_AUTO_TAKEOFF)
+    {
+        return;
+    }
+
+    if (djiStatusFlight == VehicleStatus::FlightStatus::IN_AIR)
+    {
+        velControl(setpoint);
+        return;
+    }
+
+    mVehicle->control->takeoff(mFunctionTimeout);
+}
```

Comment:
I'm starting to think that we should split the take off from the control. This way you wouldnt need to create a new `CommandAction` just to differentiate between take off with position control and with velocity control. Maybe you should just takeoff until the drone is `IN_AIR`, then leave the behavior to other controllers (e.g. ReactiveTakeoff)?

Reply:
yeah, I agree with you. I will remove the other one that is unused and then we figure out what is the best option.